rules.ini

```
@@ -221,7 +221,8 @@ NVIDIA_Ansel = (?:^|/)AnselSDK(?:32|64)\.dll$
 NVIDIA_APEX = (?:^|/)apex(?:framework|_framework|_loader|_release)
 NVIDIA_DLSS = (?:^|/)nvngx_dlss\.dll$
 NVIDIA_GeForce_Now = (?:^|/)GfnRuntimeSdk\.dll$
-NVIDIA_HairWorks = (?:^|/)GFSDK_HairWorks
+NVIDIA_HairWorks[] = (?:^|/)GFSDK_HairWorks
+NVIDIA_HairWorks[] = (?:^|/)NvHairWorksDx1(?:1|2)\.win64\.dll$
```

```suggestion
NVIDIA_HairWorks[] = (?:^|/)NvHairWorks
```
We can keep it simple